src/main/java/amazin/model/Book.java

```diff
@@ -12,47 +12,37 @@
 
 @Entity
 @Indexed
-@Table(name="book")
```

Comment:
they are not required for postgresql and add clutter to the file, it automatically stores the table Book as book columns go from camelCase to snake_case automatically, the only table I need to rename is the User table since user is a reserved keyword for postgresql